cvat-ui/src/components/actions-menu/export-submenu.tsx

```diff
@@ -23,16 +23,17 @@ export default function ExportSubmenu(props: Props): JSX.Element {
     return (
         <Menu.SubMenu key={menuKey} title='Export as a dataset'>
             {
-                exporters.map((exporter: string): JSX.Element => {
-                    const pending = (exportActivities || []).includes(exporter);
+                exporters.map((exporter: any): JSX.Element => {
+                    const pending = (exportActivities || []).includes(exporter.name);
+                    const disabled = !exporter.enabled || pending;
```

Comment:
@bsekachev, can we show disabled formats in gray?

Reply:
@zhiltsov-max What means "disabled formats"? Maybe just do not show them at all?

Reply:
Existing, but not available.